Cargo.toml

```diff
@@ -192,3 +192,7 @@ tempfile = "3.8"
 proptest = "1.0"
 proptest-derive = "0.4"
 serial_test = "2"
+
+[patch.crates-io]
```

Comment:
Will be updated once the `revm` patches have been upstreamed in https://github.com/bluealloy/revm/pull/824 and upstream reth's dep has been patched in https://github.com/paradigmxyz/reth/pull/5032